_docs-sources/guides/production-framework/ingredients/ci-cd-pipeline/intro.md

```diff
@@ -0,0 +1,7 @@
+# CI / CD Pipeline
```

Comment:
I made a similar comment in the landing zone section - and I'll generalize it here: If we've got products or guides that help a customer achieve the standards outlines in the production framework - then we should link to them. 

We can either link to the guide or link to the documentation for deploying the CI/CD pipelines. If we don't have such documentation yet for some topic - then we can consider whether we want to start tracking that and getting someone teed up to create the required content.

Reply:
Responded above